egenerator/data/handler/modular.py

```diff
@@ -343,8 +343,10 @@ def _get_data(self, file_or_frame, method, *args, **kwargs):
             assert isinstance(file_or_frame, str), "Expected file path string"
 
             num_data, data = self.data_module.get_data_from_hdf(
-                file_or_frame, *args, **kwargs
-            )
+                file_or_frame, *args,
+                label_key=self.label_module.configuration.config["label_key"],
```

Comment:
I think this needs to remain in the kwargs. Modules are not forced to specify/use a `label_key`. But I also have a slightly different implementation in the `CollectBreakingChanges` branch.

Reply:
Not quite sure that I fully understand why it hace to remain in kwargs.